apm-application-toolkit/apm-toolkit-trace/src/main/java/org/apache/skywalking/apm/toolkit/trace/IgnoredException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.toolkit.trace;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * When opening the exception status checker in the agent, a traced exception wouldn't be marked as error status which
+ * tagged with this annotation in a span. Besides the annotation supports integration, the subclasses would be also
+ * annotated with it.
```

Comment:
```suggestion
 * After the exception status checker activated in the agent, the span wouldn't be marked as error status if the exception
 * has this annotation. 
```

> Besides the annotation supports integration, the subclasses would be also annotated with it.

I think subclass status is determined by the `MAX_RECURSIVE_DEPTH`, isn't it?

Reply:
Yes, 2 ways are all controlled by MAX_RECURSIVE_DEPTH . They are almost equivalent, and both support inheritance.

org.apache.skywalking.apm.toolkit.trace.IgnoredException
org.apache.skywalking.apm.agent.core.conf.Config.StatusCheck#IGNORED_EXCEPTIONS

Reply:
Then, please correct the comment.